README.md

```diff
@@ -105,19 +105,28 @@ console.log(some_date_phase);
 ```javascript
 // print all full moons in the first quarter of 2014
```

Comment:
/facepalm

I fixed that, and while I was at it, I revised the docs to replace every instance of `new Date(YYYY, MM, DD)` with `new Date('YYYY-MM-DDTHH:MM-0500')`, so that way everyone who runs them gets the same results.

I also converted the examples to standard style.

Finally, I brought the example outputs up to date, so that way anybody who runs them will get the same results as I do. (It looks like the examples were put in before the timezone bug was fixed, and so the times were off by a few hours!)